.github/workflows/ccpp.yml

```diff
@@ -80,18 +80,34 @@ jobs:
 
     - uses: lukka/get-cmake@latest
 
-    - name: Restore from cache and run vcpkg
+    - name: 'Install CUDA'
```

Comment:
All other jobs that are CUDA-enabled have `-cuda` suffix - so I think it's best to also suffix this one with `-cuda` (i.e. `ubuntu-vcpkg-cuda`).